scripts/start-setupVanillaTweaks

```diff
@@ -33,16 +33,18 @@ if [[ "$VANILLATWEAKS_SHARECODE" ]]; then
   fi
 fi
 
-# Use vanillatweaks file to specify VT and datapacks
+# Use vanillatweaks file to specify VT and datapacks and crafting tweaks
 if [[ "$VANILLATWEAKS_FILE" ]]; then
   if [ ! -f "$VANILLATWEAKS_FILE" ]; then
     log "ERROR: given VANILLATWEAKS_FILE file does not exist"
     exit 2
   fi
 
   PACKS=$(jq -jc '.packs' $VANILLATWEAKS_FILE)
-  if [ ! "$PACKS" ]; then
-    log "ERROR: unable to retrieve packs from $VANILLATWEAKS_FILE"
+  CRAFTING_TWEAKS=$(jq -jc '.craftingtweaks' $VANILLATWEAKS_FILE)
```

Comment:
Makes me want to rename PACKS. :) Thanks for adding this!

Reply:
FWIW a technique I have done for renaming container variables is like this

https://github.com/itzg/docker-minecraft-server/blob/8a42dfe232e59163654bc0fa536f051210160e16/scripts/start-setupModpack#L190

Reply:
I think this is just my bad choice of names. :) I short handed it, not realizing it was going to get expanded out. :+1: